### PR TITLE
Adding more info about hot reload/restart on the web

### PR DESCRIPTION
**Pull request by sfshaza2**
@jonahwilliams, would you mind reviewing this small-ish change? I wanted to further clarify hot reload/restart on the web. thx

### CODE REVIEW

File: src/docs/development/tools/hot-reload.md [hunk 1]
```diff
@@ -30,6 +30,13 @@ To hot reload a Flutter app:
     If you're running the app at the command line using `flutter run`,
     enter `r` in the terminal window.
 
+{{site.alert.note}}
+  Flutter web currently supports hot restart but not
+  hot reload. When launching a web app from the command line,
+  typing `r` performs a hot restart. To perform a hot restart
+  _and_ refresh the browser, use `R`.
```
Comment (jonahwilliams) [inline src/docs/development/tools/hot-reload.md:37]: Today both `r` and `R` do exactly the same thing. We don't have support for a full restart option

Comment (sfshaza2) [reply src/docs/development/tools/hot-reload.md:37]: Oh, @jonahwilliams ! When I ran it just now, I got:

```
counter_app $ flutter run -d chrome
Launching lib/main.dart on Chrome in debug mode...
Syncing files to device Chrome...                                       
18,069ms (!)                                       
Debug service listening on ws://127.0.0.1:49847/AQ4auGO_8Bw=

Warning: Flutter's support for web development is not stable yet and
hasn't
been thoroughly tested in production environments.
For more information see https://flutter.dev/web

🔥  To hot restart changes while running, press "r". To hot restart (and
refresh the browser), press "R".
```
Is this only true on master?

```
counter_app $ flutter --version
Flutter 1.19.0-1.0.pre.43 • channel master •
ssh://git@github.com/flutter/flutter.git
Framework • revision c969b8af7b (8 hours ago) • 2020-05-11 15:28:10
-0700
Engine • revision ada8a0fd64
Tools • Dart 2.9.0 (build 2.9.0-8.0.dev a6d06b59ec)
```

Comment (jonahwilliams) [reply src/docs/development/tools/hot-reload.md:37]: Its been wrong for a while now, I opened https://github.com/flutter/flutter/pull/57450 to fix

**Comment by sfshaza2** [comment]
I want to revisit to clarify the info, but am closing for now.

**Pull request closed without merging**